sky/packages/sky/lib/base/debug.dart

```diff
@@ -4,6 +4,7 @@
 
 import 'dart:sky' as sky;
 
+/// Indicates whether we're running with asserts enabled.
```

Comment:
we really should drop this. It's redundant now that we know about `assert(() { });`.